src/monster.cpp

```diff
@@ -1923,6 +1926,14 @@ void monster::die( Creature *nkiller )
                                   pgettext( "memorial_female", "Killed a %s." ),
                                   name().c_str() );
         }
+        if( ch->is_player() && ch->has_trait( trait_KILLER ) ) {
+                std::string snip = SNIPPET.random_from_category( "killer_on_kill" );
+                if( one_in( 3 ) ) {
+                    ch->add_msg_if_player( m_good, _( snip.c_str() ) );
```

Comment:
I think this might get a bit spammy, since killing happens a lot more than lighting fires. Maybe it should only occur if you don't have the "has killed" morale boost? Just a thought.

I like it otherwise though. It fits Serial Killer, too.

Reply:
Spam is reduced by `one_in( 3 )` so 1 in 3 chances to see any snippet. If you think 33% is too much, it can be lowered to 25% or 20%.

Reply:
Ok, dropped to 25% (1 in 4).

Reply:
That should do fine. 😄